setupPRarea

```diff
@@ -0,0 +1,24 @@
+#!/bin/tcsh
+
+setenv SCRAM_ARCH slc6_amd64_gcc493
+setenv PATH /afs/cern.ch/cms/common/:$PATH
+
+#echo Renew proxy
+#voms-proxy-init -valid 100:00 -voms cms
+
+setenv pr `pwd | sed 's/.*pr//'`
```

Comment:
there should be some comments to suggest the directory structure.
It sounds like the pwd has to have "pr" in the name. ... hope that "pr" is not a part of the user name or other element in the path.
This will likely break without extra protections or a better pattern.